### PR TITLE
Location Based page

### PR DESCRIPTION
**Pull request by fcor**
- Added location based view
- Extracted file input into a Web Component

### CODE REVIEW

File: components/file-input/file-input.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        <option value="Video">Video (.mp4; max size 25MB)</option>
+      </select>
+      <a class="passive-button">Upload file</a>
+    </div>
+    <div class="pages-content-element">
+      <div class="image-placeholder">
+        Click to upload
+      </div>
+    </div>
+  </div>`;
```
Comment (fcor) [inline components/file-input/file-input.js]: How do we want to deal with styles in web components? @nicolocarpignoli

Comment (mingir2) [reply components/file-input/file-input.js]: This is how I did it due to the shadowDOM being in the way of CSS selectors. I think it works okay, but let me know if you find a better way.

https://github.com/AR-js-org/studio/pull/29/commits/cf270aedf1c00d815c318d610453e2401c11d70c

Comment (nicolocarpignoli) [reply components/file-input/file-input.js]: for me it's ok, to start this way :)

**Comment by kalwalt** [comment]
Good job ! :+1: :100:

**Comment by fcor** [comment]
I think this one is ready 😄 

I guess upload files logic would be a separate issue. If that's right, we can move on with this one.

@mingir2 @nicolocarpignoli

**Comment by nicolocarpignoli** [comment]
@fcor please align to latest modifies, I have fixed the relative paths problem, see: https://github.com/AR-js-org/studio/issues/30

**Comment by mingir2** [comment]
@nicolocarpignoli @fcor I resolved the merge conflicts and added the assetsUrl attribute to the back-anchor.

**Comment by nicolocarpignoli** [comment]
looks ok apart from the input (dropdown) style, should be align to the other on the Marker based page :)

**Pull request merged**